app/install/google-cloud.md

```diff
@@ -0,0 +1,97 @@
+---
+id: page-install-method gke-deployment
+title: Install - Google Cloud Platform
+header_title: Google Cloud Platform
+header_icon: /assets/images/icons/icn-installation.svg
+breadcrumbs:
+  Installation: /install
+links:
+  gke_repo: "https://github.com/Mashape/kong-distribution-gke"
+---
+
+Kong and its supported database can easily povisioned using the templates which can be dowloaded or cloned from following repo:
```

Comment:
As suggested, final sentence should read, `Kong can easily be provisioned to Google Cloud using the following templates:`